homeassistant/components/tplink/number.py

```diff
@@ -26,6 +27,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+@dataclass(frozen=True, kw_only=True)
```

Comment:
This missing meant the `NumberEntityDescription` was not being properly picked up hence why the `test_number` snapshot has changed from `Auto` to `Box`. Needed to be fixed in this PR in order to add the `deprecated_info` to the base `FeatureEntityDescription` without error.

Reply:
Fixed in https://github.com/home-assistant/core/pull/126397